src/main/java/cl/transbank/transaccioncompleta/FullTransaction.java

```diff
@@ -77,16 +81,31 @@ public static FullTransactionCreateResponse create(
         }
 
         public static FullTransactionCreateResponse create(
-                        String buyOrder, String sessionId, double amount, String cardNumber, short cvv, String cardExpirationDate, Options options) throws IOException, TransactionCreateException {
+                String buyOrder, String sessionId, double amount, String cardNumber, short cvv, String cardExpirationDate, Options options) throws IOException, TransactionCreateException {
             options = FullTransaction.Transaction.buildOptions(options);
             final URL endpoint = new URL(getCurrentIntegrationTypeUrl(options.getIntegrationType()));
             final WebpayApiRequest request = new TransactionCreateRequest(buyOrder, sessionId, amount, cardNumber, cvv, cardExpirationDate);
-            
+
             try {
                 return WebpayApiResource.execute(endpoint, HttpUtil.RequestMethod.POST, request, options, FullTransactionCreateResponse.class);
             } catch (TransbankException e) {
                 throw new TransactionCreateException(e);
             }
         }
+
+        public static FullTransactionInstallmentResponse installment(String token, byte installmentsNumber) throws IOException, TransactionInstallmentException {
+            return FullTransaction.Transaction.installment(token, installmentsNumber, null);
```

Comment:
Here you can use the other option of `FullTransaction.Transaction.installment` with two parameters and avoid the `null`.